src/pages/About/index.tsx

```diff
@@ -237,22 +237,37 @@ export default function About() {
             <Intro>
               <IntroCopy>The leading decentralized crypto trading protocol, governed by a global community</IntroCopy>
               <ActionsContainer>
-                <InfoButton as="a" rel="noopener noreferrer" href="https://uniswap.org" target="_blank">
-                  Learn more<WrappedExternalArrow> ↗</WrappedExternalArrow>
-                </InfoButton>
-                <InfoButton as="a" rel="noopener noreferrer" href="https://docs.uniswap.org" target="_blank">
-                  Read docs<WrappedExternalArrow> ↗</WrappedExternalArrow>
-                </InfoButton>
+                <TraceEvent
+                  events={[BrowserEvent.onClick]}
+                  name={EventName.ELEMENT_CLICKED}
```

Comment:
The event name 'Element Clicked' seems way too generic to me? could we do something like 'Landing Page Element Clicked' so at least all these relevant landing page elements are grouped together when PMs are looking through amplitude

Reply:
I talked with them and they want these to be extremely generic, since page is on there